recipes/semi-ate-spyder/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "semi-ate-spyder" %}
+{% set version = "1.0.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/semi-ate-spyder-{{ version }}.tar.gz
+  sha256: 67d71f6df0c67d035b2d991a858fd4bbecaafd35d97b4f7167fbde83f80c0c2a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - numpy
+    - pydantic
+    - python >=3.8
+    - qtawesome
+    - semi-ate-common
+    - semi-ate-plugins
+    - semi-ate-project-database
+    - semi-ate-sammy
+    - spyder >=5.3
+    - numpydoc ==1.3.1
+#    - applaunchservices >=0.1.7
+   
+    
+    
+test:
+  imports:
+    - ate_spyder
+    - ate_spyder.widgets
+#  commands:
+#    - pip check
+#  requires:
+#    - pip
```

Comment:
Could you remove these comments?

Reply:
:-)

Reply:
I meant delete the lines, just to make the recipe cleaner ;-) 

pip check does not work all the time, due to package name differences or other issues.